romanesco/executors/pyspark.py

```diff
@@ -0,0 +1,37 @@
+import imp
+import json
+import sys
+from romanesco import spark
+
+def run(task, inputs, outputs, task_inputs, task_outputs, **kwargs):
+    custom = imp.new_module("custom")
+
+    # TODO
+    # Ensure that the spark modules have been added to our path
+    # Create SparkContext - Should we share one across a workflow?
+    # Take any inputs and convert them to RDDS?
+    print 'Running pyspark'
+
+    sc = spark.create_spark_context()
```

Comment:
A note on security, just so we are all on the same page: the spark context grants access to the SparkConf that was used to initialize it, which means anyone allowed to run scripts in this mode would be able to read all of the spark configuration values very easily: `print(sc._conf.getAll())`. There's not really an easy way around this -- if we wanted to hide it, we'd have to carefully wrap the spark context object to forbid access to the configuration.

Spark apparently only allows authentication via a per-application shared secret, so this is something we might want to consider.